src/containerapp/azext_containerapp/_utils.py

```diff
@@ -1533,15 +1516,10 @@ def validate_environment_location(cmd, location):
         except Exception as e:  # pylint: disable=broad-except
             raise ValidationError("You cannot create a Containerapp environment in location {}. List of eligible locations: {}.".format(location, allowed_locs)) from e
 
-    if len(res_locations) > 0:
-        if not location:
-            logger.warning("Creating environment on location %s.", res_locations[0])
-            return res_locations[0]
-        if location in disallowed_locations:
-            raise ValidationError("You have more than {} environments in location {}. List of eligible locations: {}.".format(MAX_ENV_PER_LOCATION, location, allowed_locs))
```

Comment:
Perhaps change this validation error to mention increased quota. Something like: "You have more than {} environments in location {}. If you have an increased limit, create your containerapp using `az containerapp env create` and then pass your environment with --environment. List of eligible locations: {}."